jamo/jamo.py

```diff
@@ -287,10 +348,45 @@ def j2h(lead, vowel, tail=0):
     This function is defined solely for naming conisistency with
     jamo_to_hangul.
     """
-
     return jamo_to_hangul(lead, vowel, tail)
 
 
+def decompose_jamo(compound):
+    """Return a tuple of jamo character constituents of a compound.
+    Note: Non-compound characters are echoed back.
+
+    WARNING: Archaic jamo compounds will raise NotImplementedError.
+    """
+    if len(compound) != 1:
+      raise TypeError("decompose_jamo() expects a single character,",
```

Comment:
You can run `make lint` to check for inconsistencies like these. `flake8` will need to be installed.

Reply:
Wow, flake8 is so helpful. Rereading PEP 8 I realized how off I was. Even, my long line marker was way up at 72 for some reason...

I hacked tests to work locally yesterday by uninstalling jamo and forcing the import to look at a relative directory. I will now sheepishly read about how init files work and how to actually do this the right way though!

I'll have some updates soon, style and bugfixes the tests revealed.